src/components/Base/Upload/index.jsx

```diff
@@ -170,10 +170,9 @@ export default class Upload extends Component {
     return (
       <span
         {...events}
-        className={classNames('upload', {
+        className={classNames(className, 'upload', {
```

Comment:
className placed at last is better, since it's a override style

Reply:
ok